src/components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -54,53 +55,48 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
 
   const initialData: PatientNoteStateType = {
     notes: [],
-    cPage: 1,
-    totalPages: 1,
     facilityId: facilityId,
     patientId: patientId,
   };
   const [state, setState] = useState(initialData);
 
-  const onAddNote = async () => {
+  const { mutate: addNote } = useAddPatientNote({
+    patientId,
+    thread,
+    consultationId,
+  });
+
+  const onAddNote = () => {
     if (!/\S+/.test(noteField)) {
       Notification.Error({
         msg: "Note Should Contain At Least 1 Character",
       });
       return;
     }
-
-    const { res } = await request(routes.addPatientNote, {
-      pathParams: {
-        patientId: patientId,
-      },
-      body: {
-        note: noteField,
-        thread,
-        consultation: consultationId,
-        reply_to: reply_to?.id,
-      },
+    setReplyTo(undefined);
+    setNoteField("");
+    addNote({
+      note: noteField,
+      reply_to: reply_to?.id,
+      thread,
+      consultation: consultationId,
     });
-
-    if (res?.status === 201) {
-      Notification.Success({ msg: "Note added successfully" });
-      setState({ ...state, cPage: 1 });
-      setNoteField("");
-      setReload(true);
-      setReplyTo(undefined);
-    }
   };
 
-  useTanStackQueryInstead(routes.getPatient, {
-    pathParams: { id: patientId },
-    onResponse: ({ data }) => {
-      if (data) {
-        setPatientActive(data.is_active ?? true);
-        setPatientName(data.name ?? "");
-        setFacilityName(data.facility_object?.name ?? "");
-      }
-    },
+  const { data } = useQuery({
+    queryKey: ["/patient", patientId, thread],
```

Comment:
why do we need "thread" as a dependency for getting a patient record?
```suggestion
    queryKey: ["patient", patientId],
```

Reply:
https://github.com/ohcnetwork/care_fe/pull/9190#discussion_r1897611135
Jacob suggested me this changes

Reply:
You are always free to ask "why we need this" in-case if any requested changes didn't make sense. We might make mistakes at times, or sometimes contributors may not fully get an understanding.

Question things that doesn't make sense, instead of making changes as is requested without trying to understand.

cc: @Jacobjeevan

Reply:
Rithvik I'll take care of it from next time so sorry about it